templates/frontend/components/registrationForm.tpl

```diff
@@ -16,7 +16,7 @@
  * @uses $username string Username input entry if available
  *}
 <fieldset class="identity">
-	<legend>
+	<legend class="pkp_screen_reader">
```

Comment:
Any `<legend>`s that were visible before should remain visible. So we shouldn't add `class="pkp_screen_reader"` here or in any of the places where a `<legend>` already existed. I think you only need to add `class="pkp_screen_reader"` when you are creating a new `<legend>` just for accessibility.